ida-customizations/ida-course/exercise-week2.md

```diff
@@ -1,12 +1,26 @@
 ---
-title: "Week 2 Exercise"
+title: "Coding Exercise 1"
 order:
-description: From simple creation to practice
+description: Introducing the Final Exam
```

Comment:
The final exam already "started" in Week 1 thanks to the quiz. So perhaps "Introducing the coding part of the final exam"?